recipe/common.php

```diff
@@ -128,6 +130,22 @@
     writeln('Current release: ' . basename(get('current_path')));
 });
 
+desc('Connect to ssh and `cd` to release path');
+task('ssh', function () {
```

Comment:
@elfet could you please explain the better side of this decision? I'm pretty new in the project so I can miss something important.